bootstrap.php

```diff
@@ -11,4 +11,7 @@
 
 require 'vendor/codeigniter4/framework/app/Config/Paths.php';
 $paths = new Config\Paths();
-require_once 'vendor/codeigniter4/framework/system/bootstrap.php';
+require_once __DIR__ . '/vendor/codeigniter4/framework/system/bootstrap.php';
+
+// with this check, the class is loaded
+var_dump(class_exists(\App\Controllers\BaseController::class));
```

Comment:
is there a way to use it in rector bootstrap.php 

```php
spl_autoload_register(function (string $class): void {
    class_exists($class);
 }

```

Reply:
Yes, just:

```bash
class_exists(\App\Controllers\BaseController::class);
```

Reply:
I am trying to do with dynamic:

```php
spl_autoload_register(function (string $class): void {

    class_exists($class);
   
    // ...
});
```

that's seems overlapped, 

manually seems ok as far as it is loaded 👍 , Thank you

Reply:
I would not make this complicated, as the class is used in controllers.
Instead, I'd try get codeigniter to use PSR-4 :)